hadoop-ozone/dist/src/main/smoketest/balancer/testBalancer.robot

```diff
@@ -170,6 +177,8 @@ Verify Container Balancer for RATIS/EC containers
 
     Run Balancer Verbose Status
 
+    Sleep                   40000ms
```

Comment:
Please avoid fix amount of sleep.  Use `Wait Until Keyword Succeeds` or other retry logic.

Reply:
Thanks. Will reopen PR and fix. It seems it is flaky.

Reply:
Fixed in https://github.com/apache/ozone/pull/7569